docs/modules/data-structures/pages/updating-map-entries.adoc

```diff
@@ -278,14 +280,15 @@ Go::
 [source,go]
 ----
 // error checking is omitted for brevity
-distributedMap, _ := hzClient.GetMap("map")
+ctx := context.Background()
+distributedMap, _ := hzClient.GetMap(ctx, "map")
 entries := []types.Entry{
-	types.NewEntry(1, "John"),
+    types.NewEntry(1, "John"),
```

Comment:
I think we need to fix the indentation for the other lines as well, or revert this